src/scripts/main.js

```diff
@@ -1 +1,67 @@
 'use strict';
+
+const doc = document.querySelector('body');
+
+function showSuccses(value) {
+  doc.insertAdjacentHTML('beforeend', `
+    <div class=success data-qa="notification">${value}</div>
+    `);
+};
+
+function showError(value) {
+  doc.insertAdjacentHTML('beforeend', `
+    <div class=error data-qa="notification">${value.toString().slice(6)}</div>
+    `);
+}
+
+const firstPromise = new Promise((resolve, reject) => {
+  doc.addEventListener('click', (e) => {
+    if (e.button === 0) {
+      resolve(`First promise was resolved`);
+    } else {
+      return false;
+    }
```

Comment:
For what this `return`?
```suggestion
    }
```